features/x-provenance.rst

```diff
@@ -0,0 +1,49 @@
+.. _x-provenance:
+
+X-Provenance header
+===================
+
+Description
+-----------
+
+The X-Provenance header can be used to add a Provenance resource upon creating or updating another resource. See `Provenance#header <https://www.hl7.org/fhir/Provenance.html#header>`_ for more information.
+
+.. note:: In the case of a conditional create, where the resource was already present, the Provenance resource is not created.
```

Comment:
Probably also conditional update?

Reply:
That's actually a question I have. According to the code and https://www.hl7.org/fhir/http.html#cond-update we always do something, correct? I do know there's a 'If-Match: [etag]' or soemthing, but I can't find it in the code

Reply:
You are right, in all cases we either return an error or create a new resource. We implement the if-match header but if that doesn’t match than we return an 4xx status code